.gitignore

```diff
@@ -79,7 +79,7 @@ unlinked_spec.ds
 **/ios/**/xcuserdata
 **/ios/.generated/
 **/ios/Flutter/App.framework
-**/ios/Flutter/Flutter.framework
+**/ios/Flutter/Flutter.*
```

Comment:
It might be nice to have a comment here about why we're ignoring .*

Reply:
I could instead do:
```
**/ios/Flutter/Flutter.framework
**/ios/Flutter/Flutter.podspec
```
if you prefer.  I just figured this might give us flexibility with the next new Flutter.* thing but that might be gold-plating it.

Reply:
Ah yeah I think that would be preferable - I assumed there was some random generated garbage in there from Cocoapods.

This could end up ignoring something we don't want to...